ui/design-system-core/src/components/BasicInfoCard/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { tw } from '@twind/core';
+import BasicCardBox from '../BasicCardBox';
+import Card from '../Card';
+import Text from '../Text';
+
+interface IBasicInfoCardProps {
+  titleLabel: string;
+  subtitleLabel?: string;
+}
+
+const BasicInfoCard: React.FC<IBasicInfoCardProps> = ({ titleLabel, subtitleLabel }) => {
+  return (
+    <Card padding={8} customStyle="border-none">
+      <div className={tw('flex(& col) justify-center align-center mb-32')}>
+        <BasicCardBox style="bg-grey8 w-[180px] h-[180px] m-auto my-4" round="rounded-xl" />
```

Comment:
this card uses the same background on light and dark theme?

Reply:
fixed :))